novuhq/website
# chore: update dependencies

This pull request brings dependency update and fix few small bugs:

- updating packages include Gatsby and React
- fix z-index for mobile menu on timeline page
- small code fixes

Steps to check:

- open [preview](https://novugatsby-depsupdate.gatsbyjs.io/)
- make sure that everything looks and feels as expected
